pages/common/adb-install.md

```diff
@@ -1,4 +1,4 @@
-# adb-install
+# adb install
```

Comment:
Is this right, should it be in a separate PR just for clarity?

Reply:
The last rule in https://github.com/tldr-pages/tldr/blob/master/contributing-guides/maintainers-guide.md says:
```
Although having push access allows committing directly to the repository, please create pull requests for all of your changes, except the simplest ones (e.g. typo fixes).
```
So, I think this could be committed to `master` without reviews.

Reply:
Sure, no problem for me, was just making sure it wasn't an oversight that you might have wanted to separate.